roles/pulp_devel/tasks/main.yml

```diff
@@ -1,7 +1,10 @@
 ---
-- import_tasks: galaxy_ui.yml
+- name: include_tasks to build Galaxy WebUI if galaxy-ng is among the plugins
+  include_tasks:
+    file: galaxy_ui.yml
+    apply:
+      become: true
```

Comment:
Now I finally got why it was ignoring the `when`

Reply:
@fao89 Are you saying you ran into this problem?

And that you now understand the difference in "when to evaluate" behavior between dynamic include & static import?

Reply:
I notice it at pulplift, and yes, now I understand the difference between include/import